src/ONYXKEYS.ts

```diff
@@ -242,6 +242,7 @@ const ONYXKEYS = {
         POLICY_RECENTLY_USED_TAGS: 'policyRecentlyUsedTags_',
         WORKSPACE_INVITE_MEMBERS_DRAFT: 'workspaceInviteMembersDraft_',
         REPORT: 'report_',
+        REPORT_METADATA: 'reportMetadata_',
```

Comment:
Request: can we please add a comment to explain what belongs in `report_` and what belongs in `reportMetadata_`. Without all the context the distinction isn't very clear.

Reply:
For now we've separated the loading states `isLoadingReportActions` and `isLoadingMoreReportActions` into `reportMetadata_`, as they was causing many re-renders on smaller components every time the report actions were being fetched. It makes sense to add it a comment for future reference.

Reply:
Thanks, sounds good. Maybe the distinction is that REPORT_METADATA is for client-generated data?

Reply:
It's a performance optimization, not a hard cut between functionality/logic.